C/CellListMap/Deps.toml

```diff
@@ -10,3 +10,6 @@ ProgressMeter = "92933f4c-e287-5a05-a399-4b506db050ca"
 ["0.5.1-0"]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Setfield = "efcf1570-3423-57d1-acb7-fd33fddbac46"
+
+["0.8-0"]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
@lmiq why did you add a dependency on Documenter.jl?

[noblock]

Reply:
Thanks for noticing that, it is a mistake. But don't merge this anyway, I generated by mistake because I'm having issues with the automatic tagging of the VSCode extension. I had deleted the pull request from the repo and thought that it would suffice to block the release.